topsort-endpoints.yml

```diff
@@ -118,14 +147,31 @@ components:
         vendors:
           type: array
           description: An array of objects, each describing a vendor that should participate in the auction.
-          minItems: 0
+          minItems: 1
```

Comment:
It could actually be the case that there are no vendors.

For example one marketplace could decide to share a subset of their vendors with us. And in that case for a given category page or search results page, none of the vendors may be shared with us.

Reply:
I will remove the minItems field.